src/System.Management.Automation/System.Management.Automation.csproj

```diff
@@ -25,6 +25,7 @@
     <PackageReference Include="System.Security.AccessControl" Version="4.4.0-preview1-25302-01" />
     <PackageReference Include="System.Security.Cryptography.Pkcs" Version="4.4.0-preview1-25302-01" />
     <PackageReference Include="System.Text.Encoding.CodePages" Version="4.4.0-preview1-25302-01" />
+    <PackageReference Include="System.Threading.AccessControl" Version="4.4.0-preview1-25302-01" />
```

Comment:
It looks to me the package `System.Threading.AccessControl` is required by `Microsoft.PowerShell.Commands.Utility` instead of `System.Management.Automation`. If that's the case, this `PackageReference` entry should be added to `Microosft.PowerShell.Commands.Utility` instead. And please remove the same entry from `Microsoft.PowerShell.SDK.csproj`.